tests/ignite/contrib/handlers/test_custom_events.py

```diff
@@ -8,55 +8,44 @@
 
 def test_bad_input():
 
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent(n_iterations="a")
-
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent(n_iterations=0)
-
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent(n_iterations=10.0)
-
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent(n_epochs="a")
-
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent(n_epochs=0)
-
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent(n_epochs=10.0)
-
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent()
-
-    with pytest.raises(ValueError):
-        CustomPeriodicEvent(n_iterations=1, n_epochs=2)
+    with pytest.warns(DeprecationWarning, match=r"CustomPeriodicEvent is deprecated"):
+        with pytest.raises(ValueError):
+            CustomPeriodicEvent(n_iterations="a")
+            CustomPeriodicEvent(n_iterations=0)
```

Comment:
This is incorrect. Control flow exits `with pytest.raises(ValueError):` on the first exception. Thus, other tests are not executed at all.